site/lt_LT.all.json

```diff
@@ -1082,10 +1082,12 @@
     "other": "Kaina"
   },
   "shopping_price_plan_note": {
-    "zero": "Įkraunama kas {{ .Interval }} .",
     "one": "{{ .Interval }} kas .Intervalas pasibaigus 24 valandų nemokamam bandomajam laikotarpiui.",
```

Comment:
Lithuanian  supports [one, few, many, and other](https://github.com/nicksnyder/go-i18n/blob/0dc1626d56435e9d605a29875701721c54bc9bbd/i18n/language/codegen/plurals.xml#L164).